week2/week2Assignment.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# #Q1
+
+# In[5]:
+
+
+def fizz_buzz(input):
+    if (input % 3 == 0) and (input % 5 == 0):
+        return 'FizzBuzz'
+    if input % 3 == 0:
+        return 'Fizz'
+    if input % 5 == 0:
+        return 'Buzz'
+    return input
+print(fizz_buzz(60))
```

Comment:
here you can use else if `elif`, that would make it a bit cleaner :) as the conditions are mutually exclusive